.github/workflows/test-trezor-user-env-link-e2e.yml

```
@@ -0,0 +1,38 @@
+name: "[Test] trezor-user-env-link e2e"
+
+on:
+  pull_request:
+    branches:
+      - develop
+    paths:
+      - ".github/workflows/test-trezor-user-env-link-e2e.yml"
+      - "packages/trezor-user-env/**"
+  workflow_dispatch:
+
+concurrency:
```

actionlint (1.7.4)

group name is missing in "concurrency" section

(syntax-check)